mods/hv/maps/itrium/itrium.lua

```diff
@@ -9,7 +9,7 @@
 
 BaseBuildings = { "base", "generator", "miner2", "module" }
 
-Colonists = { Scout1, Scout2, Ballon, Miner1, Miner2, Refinery }
+Colonists = { Scout1, Scout2, Ballon, Miner1, Miner2, Storage, Generator, Refinery}
```

Comment:
You didn't name enemy actors properly so this has to be

```suggestion
Colonists = { Scout1, Scout2, Ballon, Miner1, Miner2, Actor34, Actor83, Actor84, Actor85, Refinery}
```

to work.